packages/pwa-kit-create-app/assets/templates/@salesforce/chakra-storefront/config/default.json.hbs

```diff
@@ -0,0 +1,206 @@
+{
```

Comment:
This file replaces `default.js.hbs`

Reply:
yeah let's open a ticket for that.

Reply:
The ticket, as promised:
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00002HLSCGYA5/view